scripts/configs/example_basic_gpu.yaml

```diff
@@ -0,0 +1,27 @@
+# An example configuration file.
+---
+APPROACHES:
+  my-oracle:  # used in constructing the experiment ID
+    NAME: "oracle"
+    FLAGS: {}
+  nsrt-learning:
+    NAME: "nsrt_learning"
+    FLAGS:
+      disable_harmlessness_check: True  # just an example
+ENVS:
+  cover-default-settings:  # used in constructing the experiment ID
+    NAME: "cover"
+    FLAGS: {}
+  cover-single-block:
+    NAME: "cover"
+    FLAGS:
+      cover_num_blocks: 1
+      cover_block_widths: [0.1]
+ARGS:
+  - "debug"
+FLAGS:  # general flags
+  num_train_tasks: 20
+  num_test_tasks: 10
+  use_gpu: True
```

Comment:
my preference would be to not make this a FLAG because if someone where to add the flag `--use_gpu` to a normal command line run, it would crash, so it kinda gives the wrong impression. I know that this will be more changes, but could we instead make `USE_GPU` a sibling to `START_SEED`, `NUM_SEEDS`, etc.?

Reply:
makes a ton of sense (and in fact, the current setup leads to bugs!). Will do this!